internal/api/front/perses/src/app/project/dashboard/dashboard-list/dashboard-list.component.html

```diff
@@ -0,0 +1,20 @@
+<app-page>
+  <app-page-header>
+    <h1>Dashboards</h1>
+    <button mat-raised-button color="primary">
+      <mat-icon inline>add</mat-icon>
+      New
+    </button>
+  </app-page-header>
+
+  <app-page-content>
+    <mat-spinner *ngIf="isLoading"></mat-spinner>
+    <app-page-sub-content *ngIf="!isLoading">
```

Comment:
I remember that @nexucis had a more elegant way to deal with if else in Angular in there are probably some examples in the code already. It is based on angular template as far as I remember

Reply:
the funny part is I didn't even think about it. But now you are mentioning, it would be maybe more elegant indeed. Not so sure because in the case you are mentionning @celian-garcia, it was obvious because in both cases, the HTML balise used was the same. Only the content changed. Which is not really the case, but it's worth to try it for sure !

Reply:
Some other valid cases would be when the isLoading is actually an accessor making some computation for example but yeah in that case maybe it is not. Up to you @AntoineThebaud. I can show you what we are talking about so you can see and decide.